bit_manipulation/binary_or_operator.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+https://www.tutorialspoint.com/python3/bitwise_operators_example.htm
+This function takes in 2 integer, convert them to binary and returns a binary
+in str format that is the result of an Binary OR operation from the 2 integer
+input.
+
+Returns a binary resulted from 2 integer input in str format
+>>> Binary_OR_Operator(25,32)
+'0b111001'
+
+>>> Binary_OR_Operator(37,50)
+'0b110111'
+
+>>> Binary_OR_Operator(21,30)
+'0b11111'
+
+>>> Binary_OR_Operator(58,73)
+'0b1111011'
+"""
+
+
+def Binary_OR_Operator (a : int, b : int):
+    
+    if type(a) == float or type(b) == float:

```

Comment:
Use isinstance() instead of directly comparing types as discussed in PEP8.

Reply:
alright noted, thanks